tests_tf/test_attacks.py

```diff
@@ -927,14 +814,44 @@ def test_generate_np_gives_clipped_adversarial_examples(self):
     self.assertTrue(np.max(x_adv) < .301)
 
 
-class TestMadryEtAl(CleverHansTest):
+class TestProjectedGradientDescent(CommonAttackProperties):
```

Comment:
I think this shouldn't be part of the diff. It's just checking that the empty child class can call the parent class with the right defaults and not crash.

Reply:
This is just a change of names. I made the big method called TestProjectedGradientDescent, so I had to define TestMadryEtAl